workspaces/quay/examples/entities.yaml

```diff
@@ -0,0 +1,54 @@
+---
+# https://backstage.io/docs/features/software-catalog/descriptor-format#kind-system
+apiVersion: backstage.io/v1alpha1
+kind: System
+metadata:
+  name: examples
+spec:
+  owner: guests
+---
+# https://backstage.io/docs/features/software-catalog/descriptor-format#kind-component
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: quay-site
+  annotations:
+    quay.io/repository-slug: '<organization>/<repository>'
```

Comment:
maybe add an image for this example?
```suggestion
    quay.io/repository-slug: 'rhdh-community/rhdh'
```

Reply:
Thanks, I was wondering what would be a good public repo to add here, updated: https://github.com/backstage/community-plugins/pull/2640/commits/15be2d90f13ed78f5c2451226fdf349e719bb812